hbase-it/src/main/java/org/apache/hadoop/hbase/chaos/monkies/ChaosMonkey.java

```diff
@@ -35,12 +38,14 @@
  * policy can be active at any time.
  * <p>
  * Chaos monkey can be run from the command line, or can be invoked from integration tests.
- * See {@link org.apache.hadoop.hbase.IntegrationTestIngest} or other integration tests that use
+ * See {org.apache.hadoop.hbase.IntegrationTestIngest in hbase-it /test} or other integration tests that use
```

Comment:
Is it intentional to hide the class name? Is this valid java doc?

Reply:
IntegrationTestIngest is from `src/test/java` and was inaccessible as ChaosMonkey was moved to `src/main/java`.